css/style.css

```diff
@@ -0,0 +1,2929 @@
+}
+
+.menu__link:nth-child(1) {
+  margin-left: 100px;
+  margin-right: 60px;
+  padding-right: 0;
+  padding-left: 0;
+}
+
+.menu__link:nth-child(2) {
+  padding-left: 0;
+  padding-right: 0;
+  margin-right: 18px;
+  margin-left: 0;
+}
+
+.menu__link:nth-child(3) {
+  padding-left: 0;
+  padding-right: 0;
+  margin-right: 20px;
+  margin-left: 4px;
+}
+
+.menu__link:nth-child(4) {
+  padding-right: 0;
+  padding-left: 0;
+  margin-right: 30px;
+  margin-left: 20px;
+}
+
+.menu__link:nth-child(5) {
+  padding-right: 0;
+  padding-left: 0;
+  margin-right: 98px;
+  margin-left: 12px;
+}
+
+.menu__link:hover,
+.menu__link:focus,
+.menu__link:active {
+  color: #e51937;
+  border-bottom: 2px solid #e51937;
+}
+
+.header__right {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-flow: row nowrap;
+  margin-right: 45px;
+  text-decoration: none;
+}
+
+.header__menu {
+  margin-left: 33px;
+  padding-right: 0;
+  padding-left: 0;
+}
+
+.header__menu-link {
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 18px;
+  margin-right: 30px;
+  color: #000;
+  padding-left: 0;
+  margin-right: 20px;
+  margin-left: 8px;
+  text-decoration: none;
+}
+
+.header__menu-link:nth-child(2) {
+  padding-left: 0;
+  padding-right: 0;
+  margin-right: 25px;
+}
+
+.header__menu-link:hover,
+.header__menu-link:focus,
+.header__menu-link:active {
+  width: 60px;
+  height: 2px;
+  color: #e51937;
+  border-bottom: 2px solid #e51937;
+}
+
+.header__burger {
+  padding-right: 0;
+  padding-top: 3px;
+  padding-left: 3px;
+}
+
+.top {
+  text-align: center;
+  background: url(../img/tesla-fon.png)  no-repeat;
+  background-position: 50% 0;
+  background-size: 100%;
+}
+
+.top__title {
+  max-height: 100px;
+  margin-top: 60px;
+  font-weight: 600;
+  font-size: 90px;
+  margin-bottom: 0;
+}
+
+.top__subtitle {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 18px;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.top__img {
+  max-width: 100%;
+  height: 450px;
+  margin-top: 25px;
+  margin-right: 75px;
+}
+
+.top__wrapper {
+  display: flex;
+  justify-content: center;
+  margin-top: 10px;
+  margin-left: 17px;
+  margin-bottom: 0;
+}
+
+.top__wrapper-charset {
+  display: flex;
+  align-items: center;
+  margin-right: 0;
+  margin-left: 3px;
+  margin-top: 4px;
+}
+
+.top__wrapper-item {
+  display: flex;
+  align-items: center;
+  margin-right: 100px;
+}
+
+.charset {
+  display: flex;
+  align-items: center;
+  margin-bottom: 5px;
+}
+
+.charset__item {
+  padding-right: 0;
+  padding-left: 0;
+  margin-bottom: 17px;
+}
+
+.charset__speed {
+  width: 46px;
+  height: 34.13px;
+  margin-right: 8px;
+  margin-left: 103px;
+  margin-top: 7px;
+  margin-bottom: 2px;
+}
+
+.charset__road {
+  margin-left: 25px;
+  margin-top: 15px;
+  margin-right: 0;
+}
+
+.charset__first {
+  display: flex;
+  align-items: baseline;
+  text-align-last: left;
+  font-weight: 600;
+  font-size: 35px;
+  line-height: 43px;
+  color: #000000;
```

Comment:
Codacy found an issue: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)